crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -489,6 +501,137 @@ impl CheckpointContents {
     }
 }
 
+/// Same as CheckpointContents, but contains full contents of all Transactions and
+/// TransactionEffects associated with the checkpoint.
+// TODO: rename to CheckpointContents after CheckpointContents is renamed CheckpointDigests.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct FullCheckpointContents {
```

Comment:
We have this place in the CheckpointBuilder where we estimate transaction size, can you add a comment there mentioning that this is the data structure we are trying to estimate (and similarly comment on this data structure that if it ever changes, there is code in CheckpointBuilder that needs to change).

Reply:
Thanks for the suggestion, done